frontend/apps/reader/skimtowidget.lua

```diff
@@ -359,9 +347,9 @@ end
 
 function SkimToWidget:getNextChapter(cur_pageno)
     local next_chapter = nil
-    for i = 1, #self.ticks_candidates do
-        if self.ticks_candidates[i] > cur_pageno then
-            next_chapter = self.ticks_candidates[i]
+    for _, v in ipairs(self.ticks_flattened) do
+        if v > cur_pageno then
+            next_chapter = v
```

Comment:
Can we re-use `ReaderToc:getNextChapter(cur_pageno)` ? `return self.ui.toc:getNextChapter(cur_pageno)` or even use that at the calling places.

Reply:
Oh, duh, didn't catch that among the rest of the code duplication in there, thanks ;).